docs/source/usage/installation.rst

```diff
@@ -6,6 +6,14 @@ Supported platforms
 ###################
 In order to use Mosaic, you must have access to a Databricks cluster running
 Databricks Runtime 10.0 or higher (11.2 with photon or later is recommended).
+
+.. warning::
+    From version 0.4.x, Mosaic will require either
+     * Databricks Runtime 11.2+ with Photon enabled
+     * Databricks Runtime for ML 11.2+
+
+    Non-photon clusters will not be supported any more.
```

Comment:
Can you please match the same wording as in `index.rst`?
"Other Databricks Runtime versions will not be supported anymore."